ci-operator/step-registry/stackrox/tests/groovy/stackrox-tests-groovy-commands.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+# TODO(blugo): Check if these files exist
+# shellcheck source=/dev/null
+source "scripts/ci/lib.sh"
+# shellcheck source=/dev/null
+source "tests/e2e/lib.sh"
+# shellcheck source=/dev/null
+source "scripts/ci/gcp.sh"
+# shellcheck source=/dev/null
+source "scripts/ci/sensor-wait.sh"
+# shellcheck source=/dev/null
+source "scripts/ci/create-webhookserver.sh"
+# shellcheck source=/dev/null
+source "tests/scripts/setup-certs.sh"
+# shellcheck source=/dev/null
+source "qa-tests-backend/scripts/lib.sh"
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+openshift_ci_mods
+openshift_ci_import_creds
+create_exit_trap
+
+# Might not need some of these but adding for consistency for now
+export CI_JOB_NAME="ocp-qa-e2e-tests"
+export DEPLOY_STACKROX_VIA_OPERATOR="true"
+export ORCHESTRATOR_FLAVOR="openshift"
+export ROX_POSTGRES_DATASTORE="true"
+
+gather_debug_for_cluster_under_test
+poll_for_system_test_images "3600"
+
+# Essentially replicates reuse_config_part_1
```

Comment:
Did you try setting DEPLOY_DIR to SHARED_DIR? Or creating DEPLOY_DIR again in the new context? 
And then following the same approach to executing the existing method as mentioned with config_part_1? I worry that by deconstructing and duplicating here we increase the surface for sync errors between openshift/release repo and stackrox repo.